styleguide/components/41-organisms/feedback-form/feedback-form.twig

```diff
@@ -0,0 +1,39 @@
+<div class="feedback-form {{ modifier }}">
+  <h3><button>{{ 'Can we improve something about this page?' }}<i class="icon-chevron-down" aria-hidden="true"></i></button></h3>
+  <div class="content" aria-hidden="true">
```

Comment:
Remove aria-hidden so long as the functionality is not present.
Use display:none to hide elements for all users.
(the combination with aria-hidden is correct, but I fear one day someone will only update the display property and forget to set aria-hidden to false)

Reply:
This is JS. That should belong in another future PR.

Reply:
That's my point.